### PR TITLE
Detailview controller tests

### PR DESCRIPTION
**Pull request by bart-de-jonge**
Tests for detailview, at 94% line coverage now

### CODE REVIEW

File: src/main/java/control/DetailViewController.java [hunk 1]
```diff
@@ -40,13 +39,22 @@
     private DetailView detailView;
     private ControllerManager manager;
 
+    @Getter
```
Comment (mennoo1996) [inline src/main/java/control/DetailViewController.java]: I believe in the rest of the code, we are using @Getter and @Setter on the same line. Could you conform to this standard?

**Comment by martijnjanssen** [comment]
Looks good.

**Comment by mennoo1996** [comment]
One small comment, for the rest, great work. Also, fix your build ;)

**Comment by geenen124** [comment]
Good work!

**Pull request merged**